docs/federation-reference.md

```diff
@@ -194,6 +194,35 @@ The following upstream parameters are only applicable to <a href="./federated-ex
       </td>
     </tr>
 
+    <tr>
+      <td><code>queue-type</code></td>
+      <td>
+        The queue type of the <a href="./federated-exchanges#implementation">upstream queue</a>. 
```

Comment:
Could you please use Markdown links here.

Reply:
Curiously, this indeed works with Docusaurus but did not work with the original homegrown framework. We can then replace all the links in this guide with Markdown ones.

Reply:
By "this" I mean using Markdown links in an HTML block. Obviously Markdown links work just fine in a Markdown block.